ctsit/nacculator
# Add COVID packet and update LBD Short Version

This pull request adds the COVID module to NACCulator for submission to the NACC site.
It also accounts for updates to the LBD Short Version module, adding the B2L and B6L options to the Z1X and removing fields that are no longer present in the module.

This change was made to expand NACCulator's capabilities after the changes introduced due to the pandemic and to fix a bug that was caused by NACC's DED not matching up with the information on its forms.


## Verification

_List the steps needed to make sure this thing works. Be precise._

1. Collect test COVID module data from our TESTING 1Florida ADRC REDCap project (the COVID module is present at the bottom of the data dictionary: forms "F1: COVID Technology Accessibility Survey", "F2: COVID-19 Impact Survey - Participant", and "F3: COVID-19 Impact Survey - Co-participant".
2. From the command-line, run: 
`redcap2nacc -cv <test_input.csv >test_output.txt`
3. The output should look like regular NACCulator output with the CV packet ID, ready to submit to NACC.
4. If you pass _some bad value_, the output should look like NACCulator errors printed to the command line.

5. The command for testing LBD Short Version data is:
`redcap2nacc -lbdsv -ivp <test_input.csv >test_output.txt`

- Verify the thing does this: produces COVID and LBD Short Version fixed-width text files that are ready for NACC submission.
- Verify the thing does not do that: causes KeyErrors or prints data outside of NACC's expected format.


## Affirmations

All of these should have a check by them. Any exception requires an explanation.
